workspaces/src/worker/impls.rs

```diff
@@ -0,0 +1,163 @@
+use std::collections::HashMap;
+use std::path::Path;
+
+use async_trait::async_trait;
+use near_crypto::InMemorySigner;
+use near_primitives::borsh::BorshSerialize;
+use near_primitives::types::{AccountId, Balance, FunctionArgs, StoreKey};
+
+use crate::network::{
+    Account, AllowDevAccountCreation, CallExecution, CallExecutionDetails, Contract, NetworkClient,
+    NetworkInfo, StatePatcher, TopLevelAccountCreator,
+};
+use crate::rpc::client::Client;
+use crate::worker::Worker;
+
+unsafe impl<T> Send for Worker<T> where T: Send {}
+unsafe impl<T> Sync for Worker<T> where T: Sync {}
```

Comment:
We shouldn't have this here I don't think. What could be the reason for having this instead of auto impls?

Reply:
I must have gotten rid of something that required it once upon a time. Thanks for pointing it out